ci-operator/step-registry/ipi/conf/aws/oidc-creds/ipi-conf-aws-oidc-creds-commands.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+PREFIX="${SHARED_DIR}/manifest"
+cat >> "${PREFIX}_cluster-authentication-02-config.yaml" << EOF
+apiVersion: config.openshift.io/v1
+kind: Authentication
+metadata:
+  name: cluster
+spec:
+  serviceAccountIssuer: https://<something>-oidc-discovery.s3.amazonaws.com
+
+EOF
+
+CONFIG="[default]
+role_arn = arn:aws:iam::<something>:role/<something>-cred-minter-s3-role
+web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+"
+BASE64CONFIG=$(echo ${CONFIG} | base64)
+cat >> "${PREFIX}_secret-credentials-cred-minter-s3.yaml" << EOF
+apiVersion: v1
+data:
+  credentials: ${BASE64CONFIG}
+kind: Secret
+metadata:
+  name: cloud-credential-operator-s3-creds
+  namespace: openshift-cloud-credential-operator
+
+EOF
```

Comment:
does CCO really need this permission/role when running in manual STS mode? i expected since pre-installation the S3 bucket was being created, CCO had no reason to be managing the S3 OIDC bucket.

Reply:
I don't know, if it's required or not. if it's not we can drop it. i just `oc adm release extract quay.io/openshift-release-dev/ocp-release-nightly@sha256:9663f178a9a5bf87fad0d4e2dabeaef32110d4c9c3d400eededd4f6bff5109fc --credentials-requests=true --cloud=aws --to creds` to get all the ones we need.